docs/coding.rst

```diff
@@ -283,12 +283,37 @@ This domain is simply an AWS S3 bucket with a CloudFront CDN in front of it. It
 and fast. We've made adding files to this domain very simple using `git-lfs <https://git-lfs.github.com/>`_.
 You simply install git-lfs, clone our `assets.mozilla.net repo <https://github.com/mozmeao/assets.mozilla.net>`_,
 and then add and commit files under the ``assets`` directory there as usual. Open a PR, and once it's merged
-it will be automatically uploaded to the S3 buket and be available on the domain.
+it will be automatically uploaded to the S3 bucket and be available on the domain.
 
 For example, if you add a file to the repo under ``assets/pdf/the-dude-abides.pdf``, it will be available
 as https://assets.mozilla.net/pdf/the-dude-abides.pdf. Once that is done you can link to that URL from bedrock
 as you would any other URL.
 
+Writing Migrations
+------------------
+
+Bedrock uses Django's built-in Migrations framework for its database migrations, and has no custom
+routing database routing, etc. So, no big surprises here – write things as you regularly would.
+
+*However*, as with any complex system, care needs to be taken with schema changes that
+drop or rename database columns. Due to the way the rollout process works (ask for
+details directly from the team), an absent column can cause some of the rollout to
+enter a crashloop.
+
+To avoid this, split your changes across releases, such as below.
+
+For column renames:
+
+* Release 1: Add your new column and amend the codebase to use it instead of the old column
```

Comment:
We actually need 3 steps. 

1. Add new column
2. Update code to use the new one
3. Drop the old one

This is because the database file is updated up to a few minutes after the new code is deployed, so if it expects a new column to exist the database needs to have already been updated before the code is pushed.

Reply:
I think in some situations we might be able to do 1 and 2 at the same time, but only if we can guarantee the migration will be applied before any DB-touching code is run, so the three-step plan is def safer - thanks! 👍 💅

Reply:
I don't know if we can ever know that though, due to how the migrations run and the updated DB rolls out. The DB file is sitting there on each k8s node using the previous schema when the new deployment happens. Once the clock pod is updated the migration is run and a new DB is built and uploaded to s3, but it takes some time to do that, and then for all of the updater pods on all of the nodes to check for and download the updated DB. So unless we can guarantee no traffic to whatever page or code needs the updated schema for a few minutes after the deployment then we can't really do that.